install_miniforge.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh
+# install_miniforge.sh -- install miniforge on Linux of MacOS
+
+__THIS_PATH="${0}"
+__THIS_FILE="$(basename "${__THIS_PATH}")"
+
+print_usage() {
+    echo "Install Miniforge on Linux or MacOS" 
+    echo "Usage: ${__THIS_FILE} [-h] [Miniforge3.sh arguments]"
+    # echo ""
+    # echo "(*) To install on Windows, uname, curl or wget, and sh must be installed."
+    echo ""
+    echo "If KERNEL_NAME, MARCH, and RELEASE are not specified, on this platform,"
+    echo "the defaults are:"
+    echo ""
+    echo " KERNEL_NAME=$(uname -s)"
+    echo " MARCH=$(uname -m)"
+    echo " RELEASE=latest"
+    echo ""
+    echo "## Examples:"
+    echo "### Download and run Miniforge3.sh interactively:"
+    echo "${__THIS_PATH}"
+    echo ""
+    echo "### Download Linux aarch64 latest and run Miniforge.sh interactively:"
+    echo "KERNEL_NAME=Linux MARCH=aarch64 RELEASE=latest ${__THIS_PATH}"
+    echo ""
+    echo "### Download and run non-interactively (in batch mode):"
+    echo "${__THIS_PATH} -b -p \"${HOME}/miniforge3\""
+    echo ""
+}
+
+main() {
+    for arg in "${@}"; do
+        case "${arg}" in
+            -h|--help)
+                print_usage
+                exit 0
+                ;;
+            test)
+                (set -e -x; run_tests)
+                exit $?
+                ;;
+            --install-mamba)
+                INSTALL_MAMBA=1
+                shift
+                ;;
+        esac
+    done
+    KERNEL_NAME="${KERNEL_NAME:-"$(uname)"}"
+    MARCH="${MARCH:-"$(uname -m)"}"
+    if [ "${MARCH}" = "Darwin" ]; then
+        MARCH="MacOSX"
+    fi
+
+    RELEASE="${RELEASE:-"latest"}"
+
+    MINIFORGE_URL="https://github.com/conda-forge/miniforge/releases/${RELEASE}/download/Miniforge3-${KERNEL_NAME}-${MARCH}.sh"
```

Comment:
This should be either `/latest/download/` or `/download/4.8.3-2/`. You can't have `/4.8.3-3/downloads/` nor `/download/latest` unless there's a tag with name `latest`.